tsconfig.json

```diff
@@ -10,5 +10,5 @@
   "include": [
     "src/**/*"
   ],
-  "exclude": [ "src/**/__mocks__", "test" ]
+  "exclude": [ "src/**/__mocks__", "test", "src/**/*.test.ts" ]
```

Comment:
We don't have test.ts files in src, is it really necessary as tests should be placed in the test directory ?

Reply:
There is one test now. In query-client we saved tests next to the corresponding files, and I think it is also easier to get if a file was tested or not.